tools/src/test/java/org/apache/kafka/tools/ToolsTestUtils.java

```diff
@@ -58,7 +58,7 @@ public static String captureStandardErr(Runnable runnable) {
         return captureStandardStream(true, runnable);
     }
 
-    private static String captureStandardStream(boolean isErr, Runnable runnable) {
+    public static String captureStandardStream(boolean isErr, Runnable runnable) {
```

Comment:
I don't think you need to expose this method, since every invoke of this method, the parameter `isErr` is hardcoded.
You should use another two wrapper methods (`captureStandardErr` and `captureStandardOut`) instead of direct invoking this one.